test/yapi_network_test.rb

```diff
@@ -0,0 +1,371 @@
+#! /usr/bin/env rspec
+
+require_relative "test_helper"
+
+require "yast"
+
+Yast.import "YaPI::NETWORK"
+Yast.import "NetworkInterfaces"
+Yast.import "Routing"
+Yast.import "IP"
+Yast.import "Service"
+Yast.import "DNS"
+Yast.import "LanItems"
+
+describe Yast::YaPI::NETWORK do
+  include YaPINetworkStub
+
+  describe ".Write" do
+    let(:routing) { Yast::Routing }
+
+    context "with routes" do
+      # Mock the system to do not break anything
+      before do
+        allow(routing).to receive(:Read)
+        allow(routing).to receive(:Routes)
+        allow(routing).to receive(:Write)
+      end
+
+      subject { Yast::YaPI::NETWORK }
```

Comment:
NP: You can move this line out of this context so you don't need to repeat this declaration through the whole file.